test/test_mps.py

```diff
@@ -659,8 +659,6 @@ def mps_ops_modifier(ops):
     UNIMPLEMENTED_XFAILLIST = {
         # Failures due to lack of op implementation on MPS backend
         'login': None,
-        'log_sigmoid': None,
-        'log_sigmoid_forward': None,
```

Comment:
Sorry, I'm piggybacking this one... just found they are not valid name so simply removed. The valid names should be `nn.functional.logsigmoid`